packages/cli/lib/template-listings-helper.js

```diff
@@ -0,0 +1,81 @@
+const os = require('os');
+const { join } = require('path');
+const fetch = require('isomorphic-unfetch');
+
+const fs = require('./fs');
+const { info, error } = require('./util');
+
+const {
+	TEMPLATES_REPO_URL,
+	TEMPLATES_CACHE_FOLDER,
+	TEMPLATES_CACHE_FILENAME,
+	FALLBACK_TEMPLATE_OPTIONS,
+} = require('./constants.js');
+
+async function updateTemplatesCache() {
+	const cacheFilePath = join(
+		os.homedir(),
+		TEMPLATES_CACHE_FOLDER,
+		TEMPLATES_CACHE_FILENAME
+	);
+
+	try {
+		const repos = await fetch(TEMPLATES_REPO_URL).then(r => r.json());
+		await fs.writeFile(cacheFilePath, JSON.stringify(repos, null, 2), 'utf-8');
+	} catch (err) {
+		error(`\nFailed to update template cache\n ${err}`);
+	}
+}
+
+function normalizeTemplatesResponse(repos = []) {
+	return repos.map(repo => ({
+		title: repo.name || '',
+		value: repo.full_name || '',
+		description: repo.description || '',
+	}));
+}
+
+exports.fetchTemplates = async function() {
+	let templates = [];
+	const cacheFolder = join(os.homedir(), TEMPLATES_CACHE_FOLDER);
+	const cacheFilePath = join(
+		os.homedir(),
+		TEMPLATES_CACHE_FOLDER,
+		TEMPLATES_CACHE_FILENAME
+	);
+
+	try {
```

Comment:
There are too many things that can fail in this `try...catch` forcing us to go to `FALLBACK_TEMPLATES`.
`FALLBACK_TEMPLATES` should only be rendered if there is no `cache` or it cannot be read. Otherwise always use `cache`